internal/examples/thrift-hello/hello/main.go

```diff
@@ -78,16 +108,21 @@ func do() error {
 	// using the dispatcher and associated "hello" outbound
 	client := helloclient.New(dispatcher.ClientConfig("hello"))
 
-	// build a context with a 1 second deadline
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
-	defer cancel()
+	for i := 0; i != count; i++ {
```

Comment:
can we just use `flag.Int` and use `*count` here? doesn't seem worth it to declare flags in different ways